extensions/amp-bind/0.1/test/test-bind-expression.js

```diff
@@ -268,6 +268,13 @@ describe('BindExpression', () => {
     }).to.throw(unsupportedFunctionError);
   });
 
+  it('should support encodeURI and encodeURIComponent', () => {
+    expect(evaluate('encodeURI("http://www.google.com/s p a c e.html")'))
+        .to.equal('http://www.google.com/s%20p%20a%20c%20e.html');
+    expect(evaluate('encodeURIComponent("hello world")'))
+        .to.equal('hello%20world');
```

Comment:
Nit: This test for `encodeURIComponent` tests the same functionality as the one for `encodeURI`. Recommend testing proper encoding for some of these chars: `; , / ? : @ & = + $ #`